scripts/release/generate_release_notes.sh

```diff
@@ -112,7 +112,10 @@ for cat in "${!section_titles[@]}"; do
 done
 
 for commit in "${commits[@]}"; do
-	line="- $commit ${COMMIT_METADATA_TITLE[$commit]}\n"
+	line="- $commit ${COMMIT_METADATA_TITLE[$commit]}"
+	if [[ -v COMMIT_METADATA_AUTHORS[$commit] ]]; then
+		line+=" (${COMMIT_METADATA_AUTHORS[$commit]})"
```

Comment:
Does the markdown link make them show up as a contributor in the release (this is a newish GH feature)? If not we need to use @ instead I think

Reply:
I have no idea, but I can def. revert back to @